tests/python/test-parameter-server.py

```diff
@@ -0,0 +1,368 @@
+import numpy as np
+import pinocchio as pin
+from numpy.testing import assert_almost_equal as assertApprox
+
+# Switch pinocchio to numpy matrix
+pin.switchToNumpyMatrix()
+
+import parameter_server_conf as param_server_conf
+from dynamic_graph.sot.core.parameter_server import ParameterServer
+
+param_server = ParameterServer("param_server")
+param_server.init(0.001,"rrbot.urdf","rrbot")
+
+# Control time interval
+dt=0.001
+robot_name = 'robot'
+
+# Create neutral pose for the robot
+halfSitting = [
+    0.0,
+    0.0,
+    0.0,
+    0.0
+]
+
+# Initialize configuration vector
+q = np.matrix(halfSitting).T
+print("q:")
+print(q.flatten().tolist()[0])
+
+urdfPath= param_server_conf.urdfFileName
+urdfDir= param_server_conf.model_path
+
+# Read rrbot model
+urdf_rrbot_model_string="<?xml version=\"1.0\" encoding=\"utf-8\"?>\
```

Comment:
can you use an urdf from example-robot-data, instead of hardcoding one in a string ?

Reply:
(btw, to avoid the need to escape all quotes in linebreaks, you could use triple-quotes.)

Reply:
I wanted to avoid another dependency. But this is a good idea.

Reply:
A dependency that we already have in many other projects is easier to maintain than a blob of 300 lines of code :)